### PR TITLE
update example & add judgment for comp delete

### PR DESCRIPTION
**Pull request by sunny0826**
In command `vela comp delete frontend -a frontend` , flag `-a` is necessary.

update example and add judgement on flag `-a`

Signed-off-by: guoxudong <sunnydog0826@gmail.com>

### CODE REVIEW

File: pkg/cmd/delete.go [hunk 1]
```diff
@@ -66,11 +66,18 @@ func NewCompDeleteCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Comm
...
 	cmd.SetOut(ioStreams.Out)
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
+		appName, err := cmd.Flags().GetString(App)
+		if err != nil {
+			return err
+		}
+		if appName == "" {
+			return errors.New("must specify name of application, please add flag -a")
+		}
```
Comment (wonderflow) [inline pkg/cmd/delete.go:80]: If the appName not specified, we should also possible delete component, for example use compName as AppName

File: pkg/cmd/delete.go [hunk 2]
```diff
@@ -85,10 +92,6 @@ func NewCompDeleteCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Comm
 			return errors.New("must specify name for the app")
```
Comment (wonderflow) [inline pkg/cmd/delete.go:92]:
```suggestion
			return errors.New("must specify name for the component")
```

We should change the error message

**Review by wonderflow — CHANGES_REQUESTED** [review]

File: pkg/cmd/delete.go (hunk 1 above)
Comment (sunny0826) [reply pkg/cmd/delete.go:80]: In this `delete`  appName  must specified, and the info of cmd is `Delete Component From Application`

```bash
$ bin/vela comp delete nginx
Deleting Component 'nginx' from Application ''
Error: delete appconfig err resource name may not be empty
```

File: pkg/cmd/delete.go (hunk 2 above)
Comment (sunny0826) [reply pkg/cmd/delete.go:92]:
```go
if len(args) < 1 {
			return errors.New("must specify name for the app")
		}
```
the `args` get the name of component not app , app name in flag `-a`

File: pkg/cmd/delete.go (hunk 1 above)
Comment (sunny0826) [reply pkg/cmd/delete.go:80]:
```go
appName, err := cmd.Flags().GetString(App)
if err != nil {
	return err
}
o.AppName = appName
```

the appname for flag `-a` , If the appName not specified, `err, message := o.DeleteComponent()` will throw err

**Pull request closed without merging**